src/cf-buttons/atoms/buttons.less

```diff
@@ -0,0 +1,361 @@
+//
+// Theme variables
+//
+
+// Color variables
+// $color- variables are from 18F's US Web Design Standards
+// https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
+
+// .btn
+@btn-text:                      #ffffff; // $color-white
+@btn-bg:                        #0071bc; // $color-primary
+@btn-bg-hover:                  #205493; // $color-primary-darker
+@btn-bg-active:                 #112e51; // $color-primary-darkest
+
+// .btn__secondary
+@btn__secondary-text:           #212121; // $color-base
+@btn__secondary-bg:             #9bdaf1; // $color-primary-alt-light
+@btn__secondary-bg-hover:       #02bfe7; // $color-primary-alt
+@btn__secondary-bg-active:      #00a6d2; // $color-primary-alt-dark
+
+// .btn__warning
+@btn__warning-text:             #ffffff; // $color-white
+@btn__warning-bg:               #e31c3d; // $color-secondary
+@btn__warning-bg-hover:         #cd2026; // $color-secondary-dark
+@btn__warning-bg-active:        #981b1e; // $color-secondary-darkest
+
+// .btn__disabled
+@btn__disabled-text:            darken(greyscale(#c7336e), 10%);
+@btn__disabled-bg:              lighten(greyscale(#c7336e), 40%);
+@btn__disabled-outline:         greyscale(#c7336e);
+
+// Sizing variables
+
+// .btn
+@btn-font-size:                 16px;
+@btn-v-padding:                 8px;
+@btn-v-padding-modifier-ie:     .8;
+
+// .btn__super
+@super-btn-font-size:           20px;
+
+//
+// Default button
+//
+
+.a-btn {
+
+    display: inline-block;
+    box-sizing: border-box;
+    padding:
+        unit( @btn-v-padding / @btn-font-size, em )
+        unit( 14px / @btn-font-size, em );
+    border: 0;
+    border-radius: unit( 4px / @btn-font-size, em );
+    margin: 0;
+
+    vertical-align: middle;
+
+    .webfont-medium();
+    font-size: unit( @btn-font-size / @base-font-size-px, em );
+    line-height: normal;
+    text-decoration: none;
+
+    cursor: pointer;
+    transition: background-color .1s;
+    -webkit-appearance: none;
+
+    &,
+    &:link,
+    &:visited {
+        background-color: @btn-bg;
+        color: @btn-text;
+    }
+
+    &:hover,
+    &.hover {
+        background-color: @btn-bg-hover;
+    }
+
+    &:focus,
+    &.focus {
+        background-color: @btn-bg-hover;
+        outline: 1px dotted @btn-bg;
+        // outline-offset is not supported everywhere but it adds a nice touch
+        // in browsers where it is.
+        outline-offset: 1px;
+    }
+
+    &:active,
+    &.active {
+        background-color: @btn-bg-active;
+    }
+
+    button&::-moz-focus-inner,
+    input&::-moz-focus-inner {
+        // Fixes inconsistent button.btn height in FireFox
+        // Helps with inconsistent input.btn height in FireFox but not completely
+        border: 0;
+    }
+
+    & + & {
+        margin-left: unit( 6px / @btn-font-size, em);
```

Comment:
Should this be moved to a button-group molecule? Atoms shouldn't be concerned with their surroundings.